tests/model/range.js

```diff
@@ -918,8 +918,12 @@ describe( 'Range', () => {
 		} );
 
 		describe( 'by SplitDelta', () => {
-			it( 'split inside range', () => {
+			beforeEach( () => {
+				// Cleaning...
 				root.removeChildren( root.childCount );
```

Comment:
Actually there's no need to add and remove any content from root in those tests...